### DISCUSSION TITLE
video12

### DISCUSSION THREAD

**Discussion by mcharlie9** [jacekschae/learn-pedestal-course-files#7] [Feedback]
At 3:51, "call this ApiServer": unlike a function to be called, should it be creating (or instantiating, a term you later used) a ApiServer record instead, to make the concept explanation more consistent?

At 5:13, there is a duplicated "we're just getting".

At 12:35, on why not using dissoc: would it be better to also mention that dissoc will remove the entire key-value pair; and since :service is a pre-defined field for the record, without it, the record will be turned into an ordinary map.

**Comment by jacekschae** [comment]
I reworked this a bit. The first bit is I'll keep as is, nevertheless take a note to be more precise with words. Second bit removed and cleaned additional silence. Third bit also reworked. Thanks so much! These are all great remarks!

**Reply by mcharlie9** [reply]
While reviewing the new videos, I noticed that regarding the third bit above, the codes (starting from increment 13) are indeed using dissoc instead of assoc.

**Reply by jacekschae** [reply]
I have to say that I'm very surprised by this 😱. Let me fix that. I think I have been playing around with this and committed the wrong thing. Thanks for spotting this!
